src/main/java/seedu/eduke8/question/QuizQuestionList.java

```diff
@@ -1,55 +1,56 @@
 package seedu.eduke8.question;
 
 import seedu.eduke8.common.Displayable;
-import seedu.eduke8.common.DisplayableList;
+import seedu.eduke8.exception.Eduke8Exception;
 
 import java.util.ArrayList;
 import java.util.Random;
+import java.util.logging.Logger;
+
+public class QuizQuestionList {
```

Comment:
Good idea to separate these responsibilities to two different classes

Reply:
Why does this not implement DisplayableList, maybe can change the way we think of and name this class?

Reply:
Alright I think a better fitting name would be QuizQuestionsManager as it manages the quiz questions instead of simply being a list.